packages/core/core/src/services/entity-validator/validators.ts

```diff
@@ -336,12 +337,9 @@ const addUniqueValidator = <T extends yup.AnySchema>(
     }
 
     /**
-     * If we are updating a draft and the value is unchanged we skip the unique verification. This will
-     * prevent the validator to be triggered in case the user activated the
-     * unique constraint after already creating multiple entries with
-     * the same attribute value for that field.
+     * We don't validate any unique constraint for draft entries.
      */
-    if (!isPublish && value === entity?.[updatedAttribute.name]) {
+    if (options.isDraft) {
```

Comment:
Are we agreed that uniqueness doesn't apply on drafts now? I'd be okay with that
if it's the way we want to go but when we build uniqueness for v5 we agreed on
the behaviour you can see in JIRA: DX-1422, CONTENT-2117

Reply:
yep! Our original design included draft unique validations, after doing a blitz session the PMs & Aurelien concluded drafts should not have those validations anymore.